lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -202,6 +270,30 @@ class RenderForm extends React.Component {
         supportEmail={this.props.context.supportEmail}
       />);
     }
+    if (suggestions.length) {
+      suggestionsListComponent = (
+        <ul className="suggestions">
+          {suggestions.map((suggestion, index) => {
+            let className;
+
+            if (index === activeSuggestion) {
+              className = 'suggestion-active';
+            }
+            return (
+              <li
+                className={className}
+                key={index}
+                onMouseDown={() => this.setIgnoreBlur(true)}
+                onClick={() => this.handleSuggestionClick(suggestion.url)}
+                onMouseOver={() => this.setState({ activeSuggestion: index })}
+              >
+                <div dangerouslySetInnerHTML={{ __html: suggestion.title }} />
```

Comment:
Why dangerouslySetInnerHTML? Can't we avoid it

Reply:
Each title has a string like ths: `"How do self-paced \u003cem\u003ecourses\u003c/em\u003e work?"`
which translates to "How do self-paced <em>courses</em>work?"

The keyword entered in the input field needs to be emphasized.